### ISSUE TITLE
Community meetings

### ISSUE THREAD

**Issue by rgommers** [data-apis/array-api#853]
With the array API standard development having stabilized after several releases, with more and more different projects either having implemented the standard or working on adopting its usage, and with the focus shifting to that adopting, we feel that it's time for regular public community meetings. So we're happy to start those this week!

We're also happy to announce that CZI awarded a 2 year grant as part of its EOSS program in support of work on the array API standard.🎉  That support also makes it easier to support the organising of public meetings (and whatever else seems valuable for community engagement). This new blog post has more details: https://data-apis.org/blog/eoss6_award/

These meetings will be held as Zoom calls at 6pm UTC every other Thursday, starting on 14 November. The agenda, meeting notes and call link can be found in [this HackMD document](https://hackmd.io/zn5bvdZTQIeJmb3RW1B-8g?view). If you have a topic to discuss, please feel free to add to the agenda.

**Comment by lucascolley** [comment]
will there be a meeting on the 28th?

**Comment by kgryte** [comment]
@lucascolley Yes, that is the plan.

**Comment by lucascolley** [comment]
A few points for the agenda (if there is time):
- array-api-extra and scikit-learn
	- vendor (git submodule new for scikit-learn?) vs optional dependency (not useful?) vs hard dependency (not feasible?)
- Can also link to https://github.com/data-apis/array-api/discussions/863 in the static typing section

**Comment by rgommers** [comment]
Do you have edit rights on the HackMD Lucas? If so, please go ahead and add as new agenda points directly. If not, I'll add and we can hash out the permissions on the doc together afterwards.

**Comment by lucascolley** [comment]
> Do you have edit rights on the HackMD Lucas? If so, please go ahead and add as new agenda points directly. If not, I'll add and we can hash out the permissions on the doc together afterwards.

I don't have edit right, no.

**Comment by jorenham** [comment]
It's today at 18:00 UTC, right?

**Comment by lucascolley** [comment]
> It's today at 18:00 UTC, right?

👍

**Comment by rgommers** [comment]
Hi all, Athan is ill and I'm unavailable, and we're low on topics for the next meeting (most effort for the past weeks went into finishing up v2024.12, done now 🎉). Hence we'd like to cancel this week's meeting (6 Mar).

**Comment by rgommers** [comment]
Hi all, one more cancellation for tomorrow - given a bunch of folks are at GTC or otherwise occupied, and there aren't major new topics to discuss it looks like.

**Comment by lucascolley** [comment]
in case there is time today:
- news: scikit-learn now vendors array-api-compat and array-api-extra rather than having optional dependencies: https://github.com/scikit-learn/scikit-learn/pull/30340
- Should array-api-compat support min. NumPy for scikit-learn and test it in CI? https://github.com/data-apis/array-api-compat/pull/266
- Typing and `bool | int | float | complex` https://github.com/data-apis/array-api-compat/pull/288
- CuPy multi-device support: https://github.com/data-apis/array-api-compat/pull/293

**Comment by kgryte** [comment]
Hello everyone,

We're light on topics for this week's meeting (Thursday, 26 June). So in the interest of respecting folk's time, we'd like to cancel.

@lucascolley I know you were potentially interested in discussing RNG and backward compat policies. I hope it is all right that we postpone those discussions until the next meeting on Thursday, 10 July.

Cheers!

**Comment by lucascolley** [comment]
no worries!

**Comment by kgryte** [comment]
Hey everyone,

We're back from our August recess and will be having a workgroup meeting this Thursday, 4 September. Please be sure to add any agenda topics that you'd like to discuss in the HackMD linked to in the OP above!

Cheers!

**Comment by rgommers** [comment]
Hi all, a heads up that we're skipping today's meeting, due to a combination of no new topics and unavailability of some folks that usually do the meeting prep.

**Comment by lucascolley** [comment]
is the time stated on https://hackmd.io/zn5bvdZTQIeJmb3RW1B-8g?view wrong @kgryte? Google tells me that UTC 18:00 is now but looks like the meeting has already happened!

**Comment by kgryte** [comment]
@lucascolley Yes, sorry. It does appear that the HackMD is wrong. In general, I'd subscribe to the Google calendar as the authoritative reference, as the time is US-based, not Europe-based, and with the timezone change, the time shifted for this week.

**Comment by rgommers** [comment]
Hi all, we're cancelling today's meeting, there were no new topics. I'll also be moving HackMD docs around a bit, will update links here once that is done.
